pipeline/requirements.txt

```diff
@@ -1,14 +1,15 @@
 backports-abc==0.4
 beautifulsoup4==4.5.1
```

Comment:
fingers crossed these updates are ok -- I'm experimenting with running the pipeline start to finish to make sure it doesn't cause problems prior to the brca_pseudonym_generator